File: src/entity/Item.java
package entity;

public class Item {
    private String code;
    private String name;
    private String price;
    private String qty;

    public Item() {
    }

    public Item(String code, String name, String price, String qty) {
        this.code = code;
        this.name = name;
        this.price = price;
        this.qty = qty;
    }


    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPrice() {
        return price;
    }

    public void setPrice(String price) {
        this.price = price;
    }

    public String getQty() {
        return qty;
    }

    public void setQty(String qty) {
        this.qty = qty;
    }
}

File: src/web/model/Item.js
function Item(code, name, price, qty) {

    var itemCode = code;
    var itemName = name;
    var itemPrice = price;
    var itemQty = qty;

    this.setCode = function (code) {
        itemCode = code;
    }

    this.getCode = function () {
        return itemCode;
    }

    this.setName = function (name) {
        itemName = name;
    }

    this.getName = function () {
        return itemName;
    }

    this.setPrice = function (price) {
        itemPrice = price;
    }

    this.getPrice = function () {
        return itemPrice;
    }

    this.setQty = function (qty) {
        itemQty = qty;
    }

    this.getQty = function () {
        return itemQty;
    }

}